documentation/doc-Migration_Toolkit_for_Virtualization/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -99,34 +99,38 @@ spec:
   provider:
     source:
       name: vmware-provider
-      namespace: source-namespace
+      namespace: {namespace}
     destination:
-      name: <target_cluster> <1>
-      namespace: <target_namespace> <2>
+      name: destination_cluster
+      namespace: {namespace}
   map:
-    networks:
+    networks: <1>
       - source:
-          id: <network_ref> <3>
+          id: <source_network_mor> <2>
+          name: <source_network_name> <3>
         destination:
           type: pod
           name: pod
-          namespace: <target_namespace> <4>
-    datastores:
-      - source: <5>
-          id: <datastore_ref> <6>
+          namespace: {namespace}
+    datastores: <4>
+      - source:
+          id: <source_datastore_mor> <5>
+          name: <source_datastore_name> <6>
         destination:
           storageClass: standard
   vms:
-    - id: <vm_ref_or_name> <7>
+    - id: <source_vm_mor> <7>
+    - name: <source_vm_name> <8>
 EOF
 ----
-<1> Specify the target cluster.
-<2> Specify the target namespace.
-<3> Specify the managed object reference of one or more VMware networks.
-<4> Specify the target namespace.
-<5> Specify one or more VMware data stores to map to the target data store.
-<6> Specify the managed object reference of the data store.
-<7> Specify the managed object reference or the names of one or more VMware VMs.
+<1> You can create multiple network mappings for source and destination networks.
+<2> _Managed object reference_ of the source network. If you specify the `id` parameter value, you do not need to specify the `name` parameter value.
```

Comment:
We did not test with both name & id mentioned - This may fail. 
So if we say here: 
"you do not need to..." it may cause users to leave both.
Maybe would be good to just say:
Use either id OR name.